mozilla/fx-private-relay
# Copy forwarded_first_reply values to new profile table

This PR fixes a bug introduced on the #3750 while ensuring that our local environment properly applies the new field without needlessly setting the field to accept null.

How to test:
1. Run the migration against latest main `./manage.py migrate`
2. Check the you get the error:
```
django.db.utils.IntegrityError: NOT NULL constraint failed: new__emails_profile.forwarded_first_reply
```
3. Get the changes from this PR and run migration
4. Check that you get success:
```
Running migrations:
  Applying emails.0057_profile_sent_welcome_email... OK
```
